### PR TITLE
App ws auth and remove `AppWebsocket`

### PR DESCRIPTION
**Pull request by ThetaSinner**
Note that the `AppWebsocket` isn't completely gone because it is still necessary to construct in two steps to be able to get `AppInfo` as part of construction. However, it is gone from the crate interface!

### CODE REVIEW

File: src/admin_websocket.rs [hunk 1]
```diff
@@ -74,23 +103,44 @@ impl AdminWebsocket {
...
+    /// to connect your app client.
+    ///
+    /// Allowed origins can be used to restrict which domains can connect to the interface.
+    /// This is used to protect the interface from scripts running in web pages. In development it
+    /// is acceptable to use `AllowedOrigins::All` to allow all connections. In production you
+    /// should consider setting an explicit list of origins, such as `"my_cli_app".to_string().into()`.
+    ///
+    /// If you want to restrict this app interface so that it is only accessible to a specific
+    /// installed app then you can provide the installed_app_id. The client will still need to
+    /// authenticate with a valid token for the same app but clients for other apps will not be
```
Comment (jost-s) [inline src/admin_websocket.rs:132]:
```suggestion
    /// authenticate with a valid token for the same app, but clients for other apps will not be
```

**Review by jost-s — COMMENTED** [review]
LGTM
I suggest we rename `AppAgentWebsocket` to `AppWebsocket` and use a different identifier for the now internal app websocket. What do you think?

**Comment by ThetaSinner** [comment]
Great idea, rename done!

File: src/app_agent_websocket.rs
Comment (jost-s) [inline src/app_agent_websocket.rs:1]: Rename this to app_websocket.rs?

**Review by jost-s — APPROVED** [review]

**Pull request merged**